samples/model-builder/predict_tabular_classification_sample.py

```diff
@@ -22,9 +22,17 @@
 def predict_tabular_classification_sample(
     project: str, location: str, endpoint: str, instances: List[Dict],
```

Comment:
based on the change of this PR, `endpoint` should also be changed to `endpoint_name`

Reply:
fixed